lib/matplotlib/text.py

```diff
@@ -768,7 +794,8 @@ def draw(self, renderer):
             gc.set_url(textobj._url)
             textobj._set_gc_clip(gc)
 
-            angle = textobj.get_rotation()
+            angle = (textobj.get_rotation() + (180.0*transangle/np.pi))%360.0
+
```

Comment:
Perhaps just updating the rotated 3d text unit test might be sufficient?
https://github.com/matplotlib/matplotlib/blob/master/lib/mpl_toolkits/tests/test_mplot3d.py#L128

On Tue, Mar 1, 2016 at 9:46 AM, Benjamin Root <ben.v.root@gmail.com> wrote:

> Hmm, this might be useful for Text3D objects in mplot3d.
>
> On Mon, Feb 29, 2016 at 9:53 PM, Elliott Sales de Andrade <
> notifications@github.com> wrote:
>
>> In lib/matplotlib/text.py
>> <https://github.com/matplotlib/matplotlib/pull/6081#discussion_r54516767>
>> :
>>
>> > @@ -768,7 +794,8 @@ def draw(self, renderer):
>> >              gc.set_url(textobj._url)
>> >              textobj._set_gc_clip(gc)
>> >
>> > -            angle = textobj.get_rotation()
>> > +            angle = (textobj.get_rotation() + (180.0*transangle/np.pi))%360.0
>> > +
>>
>> This extra newline is unnecessary.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/matplotlib/matplotlib/pull/6081/files#r54516767>.
>>
>
>